semantic-conventions/src/opentelemetry/semconv/model/semantic_attribute.py

```diff
@@ -275,17 +282,21 @@ def is_simple_type(attr_type: str):
         return attr_type in (
             "string",
             "string[]",
-            "number",
-            "number[]",
+            "int",
+            "int[]",
+            "double",
+            "double[]",
             "boolean",
             "boolean[]",
         )
 
     @staticmethod
     def type_mapper(attr_type: str):
```

Comment:
This name seems odd for a function. I want to suggest `parse_base_type`, but I don't know if the term "base type" is the right one for this `int[]` -> `int`, etc. mapping.